modules/caddyhttp/fileserver/browse.go

```diff
@@ -34,6 +37,18 @@ type Browse struct {
 	template *template.Template
 }
 
+const (
+	zipContentType = "application/zip"
+	zipExtension   = "zip"
+	tarContentType = "application/tar"
+	tarExtension   = "tar"
```

Comment:
Probably want `.tar.gz` instead of `.tar` (which also changes the content type)

Reply:
Hey there, what about both? Some times it is better to just use tar to avoid the cpu time compressing a gzip afterwards

Reply:
I'd be surprised if most people's download speeds were faster than the gzip compression. I'd also be surprised if most people didn't want it to be compressed when they downloaded it.

Reply:
I am more cocerned about cpu usage on the server side and some stuff won't even be compressed like a folder with pictures or movies.